docs/user-guide/rules/combine.md

```diff
@@ -2,190 +2,88 @@
 
```

Comment:
Only change in this file was moving the non-stylistic examples above the stylistic ones, i.e. leading with these two sections:

- "`*-allowed-list`, `*-disallowed-list`, `color-named` and applicable `*-no-*` rules"
- "`color-*` and `function-*` rules"